src/components/sidekick/index.tsx

```diff
@@ -36,10 +37,12 @@ export class Container extends React.Component<Properties, State> {
   static mapState(state: RootState): Partial<Properties> {
     const {
       authentication: { user },
+      directMessages: { list },
     } = state;
 
     return {
       user,
+      allUnreadMessages: list.reduce((count, directMessage) => count + directMessage.unreadCount, 0),
```

Comment:
to confirm - is this real time, or would the user have to reload to see unread count icons after receiving new message?

Reply:
we do not have realtime yet for dm's and also channels